tests/RunTests.hs

```diff
@@ -18,9 +18,6 @@ import Test.Hspec (context, hspec, it, describe, runIO, shouldSatisfy, expectati
 
 main :: IO ()
 main = hspec $ do
-  it "bazel lint" $ do
-    assertSuccess (bazel ["run", "//:buildifier"])
```

Comment:
I think this also checked files under examples and tutorial which are not covered by the new `buidifier_test` with `//:all_files`. Any way to keep those checked as well?

Reply:
I would suggest to add them as a ```local_repositories``` providing same ```all_files``` targets and use them as srcs of ```buildifier_test```

Reply:
:+1: Sounds like a good approach.

Reply:
Done